client/state/reader/teams/actions.js

```diff
@@ -7,11 +7,26 @@
  */
 import {
 	READER_TEAMS_REQUEST,
+	READER_TEAMS_RECEIVE,
 } from 'state/action-types';
 
 export function requestTeams() {
 	return {
 		type: READER_TEAMS_REQUEST,
+		meta: {
+			requestStart: READER_TEAMS_REQUEST
+		}
+	};
+}
```

Comment:
@blowery I think you're right here about the separation of concerns.

Let me catch up on what's going on in #10327, if it does provide us with a `pure and stateless descriptions of those desired HTTP requests` we should be able to implement a simple dedupe strategy on top of it without leakage.